manifests/vhost.pp

```diff
@@ -2559,7 +2559,7 @@
     concat::fragment { "${name}-serveralias":
       target  => "${priority_real}${filename}.conf",
       order   => 210,
-      content => epp('apache/vhost/_serveralias.epp', { 'serveraliases' => $serveraliases }),
+      content => epp('apache/vhost/_serveralias.epp', { 'serveraliases' => Array($serveraliases) }),
```

Comment:
This does not fix the problem as `Array('foo') #=> ['f', 'o', 'o']`.  Did you meant:

```suggestion
      content => epp('apache/vhost/_serveralias.epp', { 'serveraliases' => [$serveraliases].flatten }),
```

Reply:
I was debating that. Perhaps I was confused by Ruby's `Array()` behavior, which may also be where the bug came from.

This is why I generally prefer to only allow arrays when you want an array value.